### PR TITLE
[feat] Implement Closing All Projects Modal

### PR DESCRIPTION
**Pull request by nehaahussain**
### Description
[//]: # "[Required] Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- added a button next to the projects modal that allows user to minimize the modal
- added search bar button to open the modal when clicked
- minor fix with sort by button style lol

### Screenshots
[//]: # "Add any screenshots that will help your PR reviewer understand what’s going on in the PR!"
[//]: # "[Required] For frontend tasks: add screenshots of the layout and expected behavior (GIFs if you’re feeling extra fancy)!"
[//]: # "Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 
Screenshot 2025-03-02 at 9 53 35 PM
Screenshot 2025-03-02 at 9 53 15 PM

UPDATE:
- moved search button and all filters (status, tech, project size) down a bit so it doesn't look weird when specific project modal and search button are open at the same time.
Screenshot 2025-03-07 at 6 07 22 PM

## How to review
[//]: # "/[Required] Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"
- open & close the modal using the buttons, minimize & maximize the screen

## Next steps
[//]: # "[Optional] - What has not yet been added to the PR, doesn’t yet have any functionality — essentially, what needs to be fixed or added."
[//]: # "[Optional] any temporary fixes in this PR that should be cleaned up later."
- how should we handle the filter bar, modal and modal button styling when minimizing the width of screens or is what we have fine (shown in the photo)
Screenshot 2025-03-02 at 10 02 40 PM

### Online sources
[//]: # "[Optional] insert links to any useful tutorials or documentation that you used for this PR!"

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @jjstnlee

### CODE REVIEW

**Review by ethan-tam33 — CHANGES_REQUESTED** [review]
Looks great! I think the one issue is that the button doesn't hide the modal when the modal has a specific project opened. After this fix I think it should be good to go.

https://github.com/user-attachments/assets/a3e28175-753d-4daf-a589-e5971b1f3c3e

**Review by EthanGu4 — COMMENTED** [review]
looks good! great job neha!

File: components/ProjectsListingModal/styles.ts [hunk 1]
```diff
@@ -82,3 +83,59 @@ export const Headers = styled.div`
...
+  position: fixed;
+  left: 24rem;
+  top: 2.3rem;
+  border: none;
+  background: white;
+  cursor: pointer;
+  width: 2.375rem;
+  height: 2.875rem;
+  align-items: center;
+  border-radius: 5px;
```
[the remark this answers is not in the document]
Comment (nehaahussain) [reply components/ProjectsListingModal/styles.ts:138]: @EthanGu4 i think the other border radius styles used rem bc of an old styling sheet. i followed the new one that julee went over last work session which said to use px but good catch! maybe we can adjust the other border radius styles to px later if it really matters.

**Review by jjstnlee — APPROVED** [review]
great job neha!!!!
